### ISSUE TITLE
Better handling of ‘World’ in chart titles, annotations, and scatter plots.

### ISSUE THREAD

**Issue by mathisonian** [owid/owid-grapher#1686]
## Core problem

There are shortcomings with the way we refer to "World" currently in our charts. 

- As a label on scatter plots or line charts, it would be better to say ‘Global average’ rather than ‘World’
- In scatter plots sized by population that include both individual countries plus the world, we should consider special casing "World" so it doesn't overwhelm the other points in the chart (see e.g. https://ourworldindata.org/grapher/prevalence-of-drug-use-disorders-males-vs-females)
- We should consider improving the phrasing / grammar of "World" in chart titles (see e.g. [1](https://ourworldindata.org/grapher/disease-burden-from-injuries), [2](https://ourworldindata.org/grapher/pneumonia-mortality-by-age?country=~OWID_WRL), [3](https://ourworldindata.org/grapher/pneumonia-and-lower-respiratory-diseases-deaths) for cases where it may be better to say "Globally" vs [4](https://ourworldindata.org/grapher/share-elec-by-source) where "World" might be better).

## Proposed solution

- Labels on scatter plots or line charts say ‘Global average’ rather than ‘World’
- Scatter plots with size encoding population have a special case for the world
- Chart titles including "World" or similar are clear and grammatical

**Comment by larsyencken** [comment]
Would be improved with https://github.com/owid/owid-grapher/issues/1692

**Comment by larsyencken** [comment]
Marking this as "nice to have" because, although the wording could be improved, it's not a correctness issue. Since this is actually multiple issues, it would need breaking up before it could be worked on.

**Comment by larsyencken** [comment]
From leads discussion today, Max had a strong appetite for us to improve how we label "World" in charts. But I don't think it's ready to work on. So I'm marking it as both "important" and "needs triage".

**Comment by marcelgerber** [comment]
This is not ready to be worked on just yet, and we need to have a discussion around what this project should entail.
Daniel is gonna create an entry on the P&D board for this issue to be shaped.

**Comment by danyx23** [comment]
Shaping will happen in this [Notion Project card](https://www.notion.so/owid/Better-handling-of-World-in-charts-12174b71f92e80e89f87f53faf2e2a2f?pvs=4)

**Comment by danyx23** [comment]
We decided in issue traige that we want to go with the most specific way of solving this: adding a new config property for the `aggregateType`: `sum | average`. For now this would only work for the entity `World` and would then display this everywhere as either `Global average` or `Global total`.

Scatter plots should always show World as `Global average`

Part of this issue should also be trying to migrate as many charts as possible within a timebox of ~4 hours.

**Comment by danyx23** [comment]
We archived the Notion card in the project board now because we carved out a small and achievable version of this to live here. Should be done in cooldown or a batch cycle in Q2 2025.

**Comment by marcelgerber** [comment]
[Today on Slack](https://owid.slack.com/archives/C0149NKLZAM/p1748599029300909), a discussion came up where Simon & Tuna want to rename _any_ entities to anything else in a chart.

This is a rare use case, so we're not sure if we should ever support it.

**Comment by marcelgerber** [comment]
`aggregateType` should probably live on the indicator itself.
Then, we only apply "Global average" / "Global total" if all the indicators in a chart (minus size & color, maybe) match in terms of `aggregateType`.

**Comment by marcelgerber** [comment]
There has been a [lengthy discussion on this on Slack](https://owid.slack.com/archives/C03NA1B2P1U/p1764836294615479) today, with various concerns raised about the usefulness of this.

Crucially, we are not entirely sure whether our proposed solution of showing "Global total" / "Global average" would improve things much, and is in the realm of what max was thinking about here.
We want to hold back on this for now and check in with Max before we start working on this.
